source/core/NumericString.cpp

```diff
@@ -22,6 +22,9 @@
 
 namespace Vireo {
 
+// Encoding error codes
+enum { kInvalidUTF8CharErr = 1, kByteArrayTypeErr = 1, kCannotConvertErr = 1396 };
```

Comment:
Was double checking the error code for kInvalidUTF8CharErr and looks like desktop does not error for invalid utf8 sequences in the source byte array >.<

Seems like incorrect behavior on desktop's part. I'm not sure what ends up in the string output...